Modix.Bot/Modules/AuthorizationModule.cs

```diff
@@ -17,85 +18,84 @@ namespace Modix.Modules
     [HelpTags("claims")]
     public class AuthorizationModule : ModuleBase
     {
+        private readonly IAuthorizationService _authorizationService;
+
         public AuthorizationModule(IAuthorizationService authorizationService)
         {
-            AuthorizationService = authorizationService;
+            _authorizationService = authorizationService;
         }
 
         [Command("claims")]
-        [Summary("Lists the currently assigned claims for the current user, or a given user.")]
+        [Summary("Lists the currently assigned claims for the calling user, or given user")]
         public async Task ClaimsAsync(
-            [Summary("The user for whom to list claims, if any.")]
-                IGuildUser guildUser = null)
+
+            [Summary("User whom to list the claims for, if any")]
+            IGuildUser user = null)
+
         {
```

Comment:
Not sure how I feel about whitespace separation rather than scope-based separation. Anyone else have thoughts?

Reply:
Honestly, I thought the previous alignment/whitespacing/etc was pretty readable as it was.

Reply:
My argument for this would be readability. I'm not passionate about it, but I do feel them pushed together appears rather clustered.

Reply:
For now, this should probably be left consistent with the other modules; we can discuss the formatting for them all in a different PR.